extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -239,28 +239,27 @@ export function formatSqlCmdVariable(name: string): string {
 }
 
 /**
- * Checks if it's a valid sqlcmd variable name
+ * Checks if it's a valid sqlcmd variable name.
  * https://docs.microsoft.com/en-us/sql/ssms/scripting/sqlcmd-use-with-scripting-variables?redirectedfrom=MSDN&view=sql-server-ver15#guidelines-for-scripting-variable-names-and-values
- * @param name variable name to validate
- */
-export function isValidSqlCmdVariableName(name: string | undefined): boolean {
+ * @param cleanedName variable name to validate
```

Comment:
change param from "cleanedName" to "name" so it matches the actual parameter name.